boards/arm/stm32h7/heliguy_fmu-h747ai/configs/nsh_cm7_rptun/defconfig

```diff
@@ -9,6 +9,7 @@
 # CONFIG_NSH_DISABLE_PS is not set
 # CONFIG_STANDARD_SERIAL is not set
 # CONFIG_STM32H7_USE_LEGACY_PINMAP is not set
+CONFIG_APPS_DIR="../nuttx-apps"
```

Comment:
It's hard to change the default value base on the real apps location since it is defined in Kconfig. One possible solution is trying nuttx-apps in configure.sh/Makefile if apps doesn't exist.

Reply:
Yes, I think modifying it in the configure.sh and (C)Makefile is better

Reply:
Apologies for the day. On holidays. I've done some clean up work for the patch; it overlapped some commercial work with bespoke hardware that I'm currently working on. Hopefully the patch makes more sense now: just a standalone H747AI board with some minor H7 fixes.

Reply:
> Apologies for the day. On holidays. I've done some clean up work for the patch; it overlapped some commercial work with bespoke hardware that I'm currently working on. Hopefully the patch makes more sense now: just a standalone H747AI board with some minor H7 fixes.

Don't worry, sorry my delay too!